src/queries/redshift-schema.sql

```diff
@@ -80,6 +81,14 @@ CREATE TABLE IF NOT EXISTS rtcstats_track_metrics (
     FOREIGN KEY (pcId) REFERENCES rtcstats_pc_metrics(id)
 )
 
+/**
+ * Initial schema for redshift rtcstats_video_track table
+ */
+CREATE TABLE IF NOT EXISTS rtcstats_video_track (
```

Comment:
> Could this be represented as just another column of rtcstats_track_metrics?

@andrei-gavrilescu yep, I think you are right, I will change this. My intention was to keep the data model normalized and store all the video-related information in their own table. But that argument is compromised already since we (and by we I mean myself) already store all the frame rate and frame resolution stuff in the track details..

> Or maybe the even bigger question would be: could videoType just be extra entries for mediaType?

@nils-ohlmeier I haven't thought about it much and really just modeled this after what we already have in the client where `mediaType `and videoType are two different things. If you guys think it will be better to just have mediaType with possible values `audio`, `camera`, `screen-share`, I am happy make the change.